vault/version_store.go

```diff
@@ -149,6 +151,36 @@ func (c *Core) loadVersionHistory(ctx context.Context) error {
 	return nil
 }
 
+// isMajorVersionFirstMount compares the current version of Vault with the
+// last version that was successfully mounted and unsealed to see if it is a
+// milestone or major upgrade. It updates the core majorVersionFirstMount
```

Comment:
Stale godoc, we don't update majorVersionFirstMount here.  Do we need the field?  Could we just have setupMounts and setupCredentials call this func directly?